internal/devbox/util.go

```diff
@@ -107,3 +45,63 @@ func utilityBinPath() (string, error) {
 	}
 	return filepath.Join(nixProfilePath, "bin"), nil
 }
+
+// TODO: Commented these out so that lint doesn't complain that they're unused, but leaving them here in case we ever
```

Comment:
I think it's fine to remove. Keeping around means they will show up in grep, etc

If we ever need to bring bag, Adding a comment to PR description can make it easier to find.